### PR TITLE
Bare navbar

### PR DESCRIPTION
**Pull request by finchzebra**
A non-stylized navigation bar to be displayed at the top of all pages. The icons have click listeners, but don't yet navigate to their respective pages.

### CODE REVIEW

**Comment by jessicaawhy** [comment]
It looks like you have some commits in this PR that belong to Norman. Can you fix your branch?

**Review by xanzadu — DISMISSED** [review]

File: client/components/NavBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Logo from './Logo/Logo';
+import HomeIcon from './HomeIcon/HomeIcon';
+import LifestyleIcon from './LifestyleIcon/LifestyleIcon';
+import ShopIcon from './ShopIcon/ShopIcon';
+import Login from './Login/Login';
+
+const NavBar = () => {
+  useEffect(() => {
```
Comment (jessicaawhy) [inline client/components/NavBar/index.jsx]: I think we can remove the useEffect here!

File: client/components/NavBar/HomeIcon/HomeIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (jessicaawhy) [inline client/components/NavBar/HomeIcon/HomeIcon.jsx:1]: I'm not sure if each item will need it's own component & no styles are currently being used so we don't need to import that as well.

**Review by jessicaawhy — DISMISSED** [review]
I left a few comments to consider- feel free to address them now or later on. Approved.

File: client/components/NavBar/index.jsx (hunk 1 above)
Comment (finchzebra) [reply client/components/NavBar/index.jsx]: Thank you. Removed those lines.

File: client/components/NavBar/HomeIcon/HomeIcon.jsx (hunk 2 above)
Comment (finchzebra) [reply client/components/NavBar/HomeIcon/HomeIcon.jsx:1]: Commented out style import. Will delete later after we add style to the overall app and those lines remain unnecessary. Left each icon as its own component, but will consolidate that too, if it turns out to serve no purpose.

**Review by jessicaawhy — APPROVED** [review]

**Pull request merged**